index.d.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2019 Mark Lee and contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+declare type ChecksumOptions = {
+  defaultTextEncoding: string;
+};
+export default function sumchecker(algorithm: any, checksumFilename: any, baseDir: any, filesToCheck: any): Promise<void>;
```

Comment:
These anys seem a bit rough, can we give actual types here?

Reply:
Yep, big caveat is that I autogenerated this first :sweat_smile: